app/controllers/questions_controller.rb

```diff
@@ -32,6 +32,30 @@ def create
 
     respond_to do |format|
       if @question.save
+        # check if there are any tags for the question
+        if params[:tags_list]
+          # get names of tags the user wants to add to the question
+          tag_names_array = params[:tags_list].split(",")
+          # tags[] = array for storing Tag instances associated with the names in tag_names_array
+          tags = []
+          # loop through names from user input.
+          tag_names_array.each do |tagName|
+            # check if tag with this tagName already exists
+            if Tag.exists?(tag: tagName)
+              # get the Tag that has this tagName
+              @t = Tag.where(tag: tagName).first
+            else
+              # make a new Tag with this tagName
+              @t = Tag.new(:tag => tagName)
```

Comment:
If you do `Tag.create(:tag => tagName)` you can save a line, because the save will be implicit (create is a new plus a save)